p02_4.py

```diff
@@ -0,0 +1,36 @@
+from p02_3 import mean
+
+def stdev(arr):
+	sum = 0
+	l = len(arr)
+	if l == 0:
+		print("There are no numbers in the array!")
+		return
+	elif l == 1:
+		print("There is only one number in the array!")
+		return
+	m = mean(arr)
+	for i in range(0,l):
+		sum += (arr[i] - m) ** 2
```

Comment:
could use map() + lambda function here (which might be too much for you, but you can have a check)